pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -143,6 +144,11 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					Resources: []string{"etcds/status", "etcds/finalizers"},
 					Verbs:     []string{"get", "update", "patch", "create"},
 				},
+				{
+					APIGroups: []string{coordinationv1.GroupName},
+					Resources: []string{"leases"},
```

Comment:
Would it be better to include the `ResourceNames` to avoid giving too much permission via a `ClusterRole`?